test/functional/settlementAidOfferPage.js

```diff
@@ -225,7 +225,7 @@ settlementAidOfferPage.prototype = Object.create({}, {
     // TODO: Refactor this here and in the HTML/CSS for multiple private loans?
     privateLoanAmount: {
       get: function() {
-        return element( by.id( 'contrib__private-loan' ) );
+        return element( by.css( '[data-private-loan] [data-private-loan_key="amount"]' ) );
```

Comment:
Stay tuned for my next PR! Thrills! Chills! A bunch of functional tests for this stuff!